test/blackbox-tests/test-cases/findlib/run.t

```diff
@@ -11,3 +11,11 @@ Reproduction case for #484. The error should point to src/jbuild
   Error: Library "a" not found.
   Hint: try: dune external-lib-deps --missing @install
   [1]
+
+When passing --dev, the profile should be displayed only once (#1106):
+
+  $ jbuilder build --dev @install
```

Comment:
Can we add a test for when we use `dune` as well?

Reply:
sure!